### ISSUE TITLE
Link syntax showing in bookmark preview text

### ISSUE THREAD

**Issue by alanna** [TryGhost/Ghost#11407]
Bookmarks appear to be showing link syntax for links that appear in the preview text. To reproduce, insert a bookmark with a link in its preview text into a blog post.

<img width="758" alt="Screen Shot 2019-11-20 at 2 27 10 PM" src="https://user-images.githubusercontent.com/5498878/69201301-ae979a80-0ba2-11ea-9707-92c3e16ec0e0.png">
<img width="760" alt="Screen Shot 2019-11-20 at 2 27 00 PM" src="https://user-images.githubusercontent.com/5498878/69201302-ae979a80-0ba2-11ea-8624-d52b402e6175.png">

**Comment by gargol** [comment]
Hey @alanna! I suspect the issue here is not specifically in how Ghost fetches metadata for bookmark content but the publication's content itself. 

Could you please double-check the contents of the metadata fields (in Post Settings Menu) for these publications?  When I tried reproducing this bug for 'ZEIT' article it didn't generate markdown for the metadata, and I'm also seeing markdown output in the article itself:
![Screenshot from 2019-11-25 11-04-32](https://user-images.githubusercontent.com/675397/69511610-6c3fd600-0f73-11ea-9099-d00f019b5bd9.png)

**Comment by kevinansfield** [comment]
It's not markdown content, although it looks similar to markdown. This is how our html->plaintext conversion shows links, if you don't want links to show up in meta descriptions then in your post settings you can specify your own plain text for meta description, twitter/facebook card description, or a "custom excerpt" which will be used instead of the generated html->plaintext if it's present.

**Comment by alanna** [comment]
Thanks for the info. I added a custom excerpt and also put the same text in the meta data description, then re-published. However, it's still showing the link markup syntax in the bookmark on the other post which refers to it. Does it usually take some time for changes to propagate?

**Comment by gargol** [comment]
> Does it usually take some time for changes to propagate?

**TLDR**: should be propagated immediately if you have default setup.

The longer answer, this depends on the caching server if you have something custom in place. Internal `X-Cache-Invalidate` cache headers are being reset in Ghost if you republish the post - https://github.com/TryGhost/Ghost/blob/3.1.0/core/server/api/canary/posts.js#L173
and later https://github.com/TryGhost/Ghost/blob/3.1.0/core/server/api/shared/headers.js#L9.

And the `Cache-Control` header depends on your config, but usually is set to: `Cache-Control: max-age=0` [here](https://github.com/TryGhost/Ghost/blob/3.1.0/core/server/web/shared/middlewares/cache-control.js#L14) which tells the browser it should revalidate the response. 

Can you confirm if the cache propagated for you?

**Comment by gargol** [comment]
@alanna feel free to reopen this issue again if you think there is a bug still. Closing it, for now, to keep the issues tidy :wink:
